src/main/java/io/vertx/core/eventbus/impl/clustered/ClusterNodeInfo.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2011-2017 The original author or authors
+ * ------------------------------------------------------
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ *
+ *     The Eclipse Public License is available at
+ *     http://www.eclipse.org/legal/epl-v10.html
+ *
+ *     The Apache License v2.0 is available at
+ *     http://www.opensource.org/licenses/apache2.0.php
+ *
+ * You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.core.eventbus.impl.clustered;
+
+import java.io.Serializable;
+
+/**
+ * @author Rikard Björklind
+ */
+public class ClusterNodeInfo implements Serializable {
```

Comment:
can we have ClusterNodeInfo extends or embed ServerID ? that would avoid to recreate a ServerId from ClusterNodeInfo later